PlaygroundService/Controllers/PlaygroundController.cs

```diff
@@ -22,6 +23,25 @@ public PlaygroundController(IClusterClient client, ILogger<PlaygroundController>
             _client = client;
             _logger = logger;
         }
+        
+        [HttpGet("templates")]
+        public async Task<IActionResult> GetTemplateConfig()
+        {
+            _logger.LogInformation("templates  - GetTemplateConfig started time: "+ DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss"));
+            var templateConfGrain = _client.GetGrain<IPlaygroundGrain>("userId");
```

Comment:
Currently using the same grain over and over. Will need to insert the real userId instead.

Reply:
Okay, the next step is to add the user ID